src/ngParseExt/module.js

```diff
@@ -6,6 +6,12 @@
  * @ngdoc module
  * @name ngParseExt
  * @packageName angular-parse-ext
+ *
+ * @knownIssue
+ * You cannot use Unicode characters for variable names in the {@link ngRepeat} or {@link ngOptions}
+ * expressions (e.g. `ng-repeat="f in поля"`), because they don't use the {@link `$parse`} service
+ * to parse the expression.
```

Comment:
I think this needs a bit of rewriting to be clear.

> You cannot use Unicode characters for variable names in the {@link ngRepeat} or {@link ngOptions} expressions (e.g. `ng-repeat="f in поля"`). This can be surprising because the {@link `$parse`} service can recognize these characters but these expressions are not parsed by `$parse`.